lua/nvim-tree/utils.lua

```diff
@@ -11,6 +11,14 @@ function M.echo_warning(msg)
   api.nvim_command('echohl None')
 end
 
+function M.read_file(path)
```

Comment:
i'd prefer using `vim.loop` here:
[docs](https://github.com/luvit/luv/blob/master/docs.md)
code from documentation:
```
local function readFileSync(path)
  local fd = assert(uv.fs_open(path, "r", 438))
  local stat = assert(uv.fs_fstat(fd))
  local data = assert(uv.fs_read(fd, stat.size, 0))
  assert(uv.fs_close(fd))
  return data
end
```
without the asserts ;)
you could access `uv` as `vim.loop` (i like to call it `luv`)